potpie/context-engine/adapters/inbound/cli/templates/claude_plugin/commands/potpie-feature.md

```
@@ -0,0 +1,23 @@
+Use Potpie context before feature work. Load `potpie-project-preferences` first,
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)